### PR TITLE
translation: Update chapter_computational_complexity/performance_evaluation.md

### PR DESCRIPTION
**Pull request by Phoenix0415**
If this PR is related to coding or code translation, please fill out the checklist and paste the console outputs to the PR.

- [ ] I've tested the code and ensured the outputs are the same as the outputs of reference code.
- [ ] I've checked the code (formatting, comments, indentation, file header, etc) carefully.
- [ ] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

**Comment by krahets** [comment]
Pending 3 reviewers’ approvals.

File: docs-en/chapter_computational_complexity/performance_evaluation.md [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+Complexity analysis provides us with a 'ruler' for evaluating the efficiency of algorithms, enabling us to measure the time and space resources required to execute a given algorithm and to compare the efficiency of different algorithms.
 
-Complexity is a mathematical concept that may be abstract and relatively difficult for beginners to learn. From this point of view, complexity analysis may not be very suitable as the first introduction. However, when we discuss the characteristics of a particular data structure or algorithm, it is difficult to avoid analyzing its running speed and space usage.
+Complexity is a mathematical concept that might seem abstract and somewhat challenging for beginners. From this perspective, introducing complexity analysis at the very beginning may not be the most suitable approach. However, when discussing the characteristics of a particular data structure or algorithm, analyzing its operational speed and space usage is often unavoidable.
```
Comment (thomasq0) [inline docs-en/chapter_computational_complexity/performance_evaluation.md:46]: inevitable v.s. unavoidable?

**Review by yuelinxin — COMMENTED** [review]
Some small tweaks should do it.

File: docs-en/chapter_computational_complexity/performance_evaluation.md [hunk 2]
```diff
@@ -1,48 +1,49 @@
...
+1. **Finding a Solution to the Problem**: The algorithm needs to reliably find the correct solution within the specified input range. This is our initial, more straightforward objective.
+2. **Seeking the Optimal Solution**: There may be multiple ways to solve the same problem, and our goal is to find the most efficient algorithm possible. This is the more advanced and challenging goal.
 
-That is to say, under the premise of being able to solve the problem, algorithm efficiency has become the main evaluation index for measuring the merits of algorithms, which includes the following two dimensions.
+In other words, once the ability to solve the problem is established, the efficiency of the algorithm emerges as the main benchmark for assessing its quality, which includes the following two aspects.
 
-- **Time efficiency**: how fast or slow the algorithm runs.
-- **Space efficiency**: the amount of memory space occupied by the algorithm.
+- **Time Efficiency**: The speed at which an algorithm runs.
+- **Space Efficiency**: The amount of memory space the algorithm occupies.
```
[the remark this answers is not in the document]
Comment (Phoenix0415) [reply docs-en/chapter_computational_complexity/performance_evaluation.md:11]: very good advice, will update it in the  next PR

File: docs-en/chapter_computational_complexity/performance_evaluation.md [hunk 3]
```diff
@@ -1,48 +1,49 @@
...
 
-In short, our goal is to design data structures and algorithms that are "fast and frugal". Evaluating the efficiency of algorithms effectively is crucial because only then can we compare various algorithms to guide the algorithm design and optimization process.
+In short, our goal is to design data structures and algorithms that are both "fast and economical". Effectively evaluating algorithm efficiency is crucial, as it allows for the comparison of different algorithms and guides the design and optimization process.
 
-There are two main types of efficiency assessment methods: practical testing, and theoretical estimation.
+There are mainly two methods for assessing efficiency: practical testing and theoretical estimation.
```
[the remark this answers is not in the document]
Comment (Phoenix0415) [reply docs-en/chapter_computational_complexity/performance_evaluation.md:16]: Agree, will update it in the next PR.

**Comment by Phoenix0415** [comment]
@thomasq0 @yuelinxin Thank you so much for your feedback! I completely agree with your suggestions and have incorporated them into the new commits.

**Review by yuelinxin — APPROVED** [review]
LGTM! Thank you for your kind contribution @Phoenix0415

**Review by thomasq0 — APPROVED** [review]

**Review by krahets — APPROVED** [review]
Thanks @Phoenix0415 for the PR and throughout review from @yuelinxin @thomasq0 .

**Pull request merged**